syntax/std_runtime.go

```diff
@@ -1,14 +1,34 @@
 package syntax
 
 import (
-	"strings"
+	"fmt"
 
 	"github.com/arr-ai/arrai/rel"
 )
 
 // BuildInfo represents arr.ai build information.
 var BuildInfo rel.Value
 
+const buildInfoTemplate = `
+(
+    version: '%s',
+    date: '%s',
+    git: (
+        commit: '%s',
+        tags: ['%s'],
+    ),
+    go: (
+        os: '%s',
+        arch: '%s',
+        compiler: (
+            version: '%s',
+            os: '%s',
+            arch: '%s',
+        ),
+    ),
+)
```

Comment:
Pasting together arr.ai syntax with `fmt.Sprintf` is a code smell. The original code was preferable.